### PR TITLE
Recal isopeaks error

### PR DESCRIPTION
**Pull request by KristinaGomoryova**
This PR resolves https://github.com/RECETOX/MFAssignR/issues/45

The Recal function was failing when `isopeaks = "none"`

Problem was in line
`isopeaks <- ifelse(isopeaks == "none", data.frame(mass = 1, Abundance = 1, Tag = "X"), isopeaks)` 
when instead of dataframe a list was created.

This was solved as following:
`isopeaks <- if(isopeaks == "none"){data.frame(mass = 1, Abundance = 1, Tag = "X")}else{isopeaks}`

### CODE REVIEW

File: R/RecalFunctions.R [hunk 1]
```diff
@@ -178,8 +179,11 @@ Recal <- function(df,
   df$mode <- mode
   names(df)[1:2] <- c("Abundance", "Exp_mass")
 
+  O_mass <- 15.9949146223
+  H_mass <- 2.01565
```
Comment (hechth) [inline R/RecalFunctions.R:183]: The mass of hydrogen is for sure not 2! Should be renamed to H2_mass I assume?

File: R/RecalFunctions.R [hunk 2]
```diff
@@ -286,34 +290,9 @@ Recal <- function(df,
...
-    Step3$H2_num <- round(((Step3$Exp_mass - Step3$base_mass)) / 2.01565)
-    Step3$H <- Step3$H + 2 * Step3$H2_num
-    Step3$Type <- "H2"
-    Step3$form <- paste(Step3$C, Step3$H, Step3$O, Step3$N, Step3$S, Step3$P, Step3$E, Step3$S34,
-      Step3$N15, Step3$D, Step3$Cl, Step3$Fl, Step3$Cl37, Step3$M, Step3$NH4,
-      Step3$POE, Step3$NOE,
-      sep = "_"
-    )
-    Step3 <- Step3[abs(Step3$H2_num) <= step_H2, ]
-    Step3 <- Step3[-c(11, 13, 33)]
```
Comment (hechth) [inline R/RecalFunctions.R:316]: How come that this is all no longer needed?

File: R/RecalFunctions.R (hunk 1 above)
Comment (KristinaGomoryova) [reply R/RecalFunctions.R]: resolved in 7992e2830b8d6611823f05e6d02f750e0dab97fe

File: R/RecalFunctions.R (hunk 2 above)
Comment (hechth) [reply R/RecalFunctions.R:316]: test

Comment (KristinaGomoryova) [reply R/RecalFunctions.R:316]: teat

**Pull request closed without merging**